src/controller/signup.controller.ts

```diff
@@ -8,7 +8,12 @@ export default {
       const id = await serviceSignup.createUser(req.body);
       res.status(201).send(id);
     } catch (error) {
+      let message;
+      if (error instanceof Error) {
+        message = error;
+        res.status(400).send(message.toString());
```

Comment:
Should we be sending an explicit error message to the frontend here? Maybe a generic message is better in this case.